src/orcania.h

```diff
@@ -10,6 +10,10 @@
 #ifndef __ORCANIA_H__
 #define __ORCANIA_H__
 
+#if defined(__MINGW32__) || defined(__CYGWIN__)
```

Comment:
Does the `defined(__CYGWIN__)` mean it would run the same on Cygwin as well? If not I'd rather not imply that Orcania can be built on Cygwin.

Reply:
You are absolutely right. I (and most of C programmers) don't use Cygwin anymore, so we need to keep only the MinGW building. I'm going to remove all `__CYGWIN__` defines ...